### DISCUSSION TITLE
Crystal orientation definition for statistically equivalent RVE

### DISCUSSION THREAD

**Discussion by gomezbenavente** [neperfepx/neper#865] [Q&A] [answered]
Hi,

I have a dataset of grain orientations (specifically, using the Euler angles description), from which I can obtain a 3-variate normal distribution that is representative of that dataset. If I want to generate a _statistically equivalent RVE_ in terms of orientations, **is there a way to introduce the parameters of my distribution into Neper as an orientation distribution function (ODF)** so that they are taken into account when generating the RVE?

I have tried using the -ori command, but I can't understand why I have to indicate as well a "discrete orientation" in addition to the ODF, because the Euler angles on the dataset are defined w.r.t. the global frame of reference.

Thank you in advance.
Kind regards,

**Comment by rquey** [comment, marked as answer]
In `-ori`, the 3-variate normal distribution is defined about a nominal orientation, as in `Goss:"normal(theta=...)"`.
